substra/cli/printers.py

```diff
@@ -167,30 +167,35 @@ def _get_list_fields(self):
     def _get_single_fields(self):
         return (self.key_field, ) + self.single_fields
 
-    def print_download_message(self, item):
+    def print_with_profile(self, profile):
+        if profile and profile != 'default':
+            return f'--profile {profile}'
+        return ''
+
+    def print_download_message(self, item, profile=None):
         if self.download_message:
             key_value = self.key_field.get_value(item)
-            print()
-            print(self.download_message)
-            print(f'\tsubstra download {self.asset_name} {key_value}')
+            profile_arg = self.print_with_profile(profile)
+            print('\n' + self.download_message)
+            print(f'\tsubstra download {self.asset_name} {key_value} {profile_arg}')
 
-    def print_description_message(self, item):
+    def print_description_message(self, item, profile=None):
         if self.has_description:
             key_value = self.key_field.get_value(item)
-            print()
-            print(f'Display this {self.asset_name}\'s description:')
-            print(f'\tsubstra describe {self.asset_name} {key_value}')
+            profile_arg = self.print_with_profile(profile)
+            print('\n' + f'Display this {self.asset_name}\'s description:')
```

Comment:
No need to concatenate strings here, you can write `f'\nDisplay etc.'`.

Reply:
Yes, I'll change that